docs/index.rst

```diff
@@ -0,0 +1,54 @@
+.. Licensed under the BSD-3-Clause: https://opensource.org/licenses/BSD-3-Clause
+
+===========
+udiff
+===========
+
+``udiff`` is a tool for automatic differentiation with uarray/unumpy.
+
+.. warning::
+
+   Working in progress. This is a pre-release build.
+
+Quick start
+-----------
+
+Getting started is easy:
+
+#.  Use Git to clone the ``udiff`` repository::
+
+        $ git clone https://github.com/Quansight-Labs/udiff.git
+        $ cd udiff
+
+#.  Install ``udiff`` on the command line, enter::
+
+        $ python setup.py install --user
+
+#.  Use ``udiff`` in your code. A simple example::
+
+         import uarray as ua
```

Comment:
We can't test these examples. 😦 And the docs may go out of date. Can you add a docstring to the `udiff` module and use autodoc?

Docstring example: https://github.com/Quansight-Labs/unumpy/blob/master/unumpy/__init__.py
Autodoc: https://github.com/Quansight-Labs/unumpy/tree/master/docs/generated

Reply:
Fine. I'll try.

Reply:
Hi, I made some changes, could you please take a look?